includes/admin/reporting/reports.php

```diff
@@ -2545,335 +2544,424 @@ function display_export_report() {
 										'chosen'      => true,
 										/* translators: the plural post type label */
 										'placeholder' => sprintf( __( 'All %s', 'easy-digital-downloads' ), edd_get_label_plural() ),
 									)
-								); ?>
-								<?php wp_nonce_field( 'edd_ajax_export', 'edd_ajax_export' ); ?>
-								<input type="hidden" name="edd-export-class" value="EDD_Batch_Downloads_Export"/>
-								<button type="submit" class="button button-secondary"><?php esc_html_e( 'Generate CSV', 'easy-digital-downloads' ); ?></button>
-                            </form>
-                        </div>
-                    </div>
-
-                    <div class="postbox edd-export-api-requests">
-                        <h3 class="hndle"><span><?php esc_html_e('Export API Request Logs','easy-digital-downloads' ); ?></span></h3>
-                        <div class="inside">
-                            <p><?php esc_html_e( 'Download a CSV of API request logs.', 'easy-digital-downloads' ); ?></p>
-                            <form id="edd-export-api-requests" class="edd-export-form edd-import-export-form" method="post">
-								<div class="edd-from-to-wrapper">
-					                <?php
-
-									echo EDD()->html->date_field( array(
-										'id'          => 'edd-api-requests-export-start',
-										'class'       => 'edd-export-start',
-										'name'        => 'start',
-										'placeholder' => _x( 'From', 'date filter', 'easy-digital-downloads' )
-									) );
+								);
 
-									echo EDD()->html->date_field( array(
-										'id'          => 'edd-api-requests-export-end',
+								wp_nonce_field( 'edd_ajax_export', 'edd_ajax_export' );
+								?>
+							<input type="hidden" name="edd-export-class" value="EDD_Batch_Customers_Export"/>
+							<button type="submit" class="button button-secondary"><?php esc_html_e( 'Generate CSV', 'easy-digital-downloads' ); ?></button>
+						</form>
+					</div>
+				</div>
+
+				<div class="postbox edd-export-taxed-customers">
+					<h2 class="hndle"><span><?php esc_html_e( 'Export Taxed Customers', 'easy-digital-downloads' ); ?></span></h2>
+					<div class="inside">
+						<p><?php esc_html_e( 'Download a CSV of all customers that were taxed.', 'easy-digital-downloads' ); ?></p>
+						<form id="edd-export-taxed-customers" class="edd-export-form edd-import-export-form" method="post">
+							<fieldset class="edd-from-to-wrapper">
+							<legend class="screen-reader-text">
+								<?php esc_html_e( 'Export Taxed Customer Dates', 'easy-digital-downloads' ); ?></legend>
+							<label for="edd-taxed-customers-export-start" class="screen-reader-text"><?php esc_html_e( 'Set start date', 'easy-digital-downloads' ); ?></label>
+								<?php
+									echo EDD()->html->date_field(
+										array(
+											'id'          => 'edd-taxed-customers-export-start',
+											'class'       => 'edd-export-start',
+											'name'        => 'taxed-orders-export-start',
```

Comment:
This is actually taxed customers, not orders (creating a duplicate ID).

```suggestion
											'name'        => 'taxed-customers-export-start',
```